corehq/form_processor/migrations/0073_drop_case_uuid_like_index.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-03 20:44
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db.migrations import Migration
+
+from corehq.sql_db.operations import HqRunSQL
+
+
+class Migration(Migration):
+
```

Comment:
I think that you want to specify `atomic=False` if you want to do this concurrently  https://github.com/dimagi/commcare-hq/pull/17527